FirebaseCrashlytics.podspec

```diff
@@ -55,6 +55,7 @@ Pod::Spec.new do |s|
 
   s.dependency 'FirebaseCore', '~> 10.0'
```

Comment:
This needs to be at least 10.1 since Sessions support was added to Core after 10.0. Recommend making this 10.5 (or 10.6 after the podspecs are updated later this week)